src/target/riscv/riscv.c

```diff
@@ -5568,8 +5568,7 @@ static enum riscv_halt_reason riscv_halt_reason(struct target *target)
 
 size_t riscv_progbuf_size(struct target *target)
 {
-	RISCV_INFO(r);
-	return r->progbuf_size;
+	return riscv_info(target)->get_progbufsize(target);
```

Comment:
(same as in the previous comment)

I'd prefer RISCV_INFO() for consistency with the other functions around.

```suggestion
	RISCV_INFO(r);
	return r->get_progbufsize(target);
```

Reply:
Addressed